lib/wallaroo/core/router_registry/router_registry.pony

```diff
@@ -786,9 +806,22 @@ actor RouterRegistry
     end
 
   fun ref try_to_resume_processing_immediately() =>
-    if _key_waiting_list.size() == 0 then
+    let ar' = recover iso Array[U8] end
```

Comment:
If this is for debugging can you mark it somehow so you don't miss it later.  You could lead with
```
// [jb]
```
If you use that to find things that need to be removed.